src/sharding/ShardingManager.js

```diff
@@ -19,8 +19,9 @@ const Util = require('../util/Util');
  */
 class ShardingManager extends EventEmitter {
   /**
-   * The mode to spawn shards with for a {@link ShardingManager}. Either "process" to use child processes, or
-   * "worker" to use [Worker threads](https://nodejs.org/api/worker_threads.html).
+   * The mode to spawn shards with for a {@link ShardingManager}. Can be either one of
+   * * 'process' to use child processes
+   * * 'worker' to use [Worker threads](https://nodejs.org/api/worker_threads.html).
```

Comment:
```suggestion
   * The mode to spawn shards with for a {@link ShardingManager}. Here are the available modes:
   * * 'process' to use child processes
   * * 'worker' to use [Worker threads](https://nodejs.org/api/worker_threads.html)
```

Reply:
This suggested wording feels rather informal and not right for the docs

Reply:
Yeah I used the “format” that is generally used across the docs

Reply:
Idk, I saw a bunch of stuff like `Here are the available types:` which is why I suggested it

Reply:
It does exist, but the current version appears more often

Reply:
Alright, just colon then